packages/date/src/conversion.ts

```diff
@@ -14,38 +14,38 @@ import {LocalDate, LocalDateTime, nativeJs, ZonedDateTime, ZoneId, LocalTime} fr
  */
 
 export interface ILocalDateLike {
-	_year: number;
-	_month: number;
-	_day: number;
+	year: number;
+	month: number;
+	day: number;
```

Comment:
The internal structure of a LocalDate uses underscores, so they have to remain here too.